app/services/pickup_rate/task.py

```
@@ -0,0 +1,82 @@
+)
+from app.core.config.static import SLACK_WEBHOOK_URL
+from app.core.logger import logger
+from app.services.pickup_rate.monitor import pickup_rate_monitor
+
+
+async def initialize_pickup_rate_tasks(scheduler) -> bool:
+    """
+    Register pickup rate alert tasks if all required configuration is present.
+
+    Registers two independent tasks:
+    1. pickup_rate_monitor            – global alert, gated by ENABLE_PICKUP_RATE_ALERT
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)